packages/ui/src/stories/DashboardShowcase.stories.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect } from 'react'
+import { BrowserRouter } from 'react-router-dom'
+import styled from 'styled-components'
+
+import { Button } from '../Button'
+import { Icon } from '../Icon'
+import { Page, PageLayout, usePageLayout } from '../Layouts'
+import { GridLayout } from '../Layouts/GridLayout'
+import { Table } from '../Table'
+import { Text } from '../Text'
+
+import { defaultColumns, defaultData } from './Table.stories'
+
+export default {
+  title: 'Showcases/Dashboard',
+  component: Page,
+  argTypes: {
+    header: {
+      defaultValue: true,
```

Comment:
default values are being deprecated, now we should use

Overview.args = {header: true}

Reply:
Updated :)